README.md

```diff
@@ -50,7 +50,28 @@ python3 -m gpucsl --gaussian -d ./data.csv -o . -l 3
 - `discover_skeleton_gpu_discrete` - determines the undirected skeleton graph for discrete distribution
 - `orient_edges` - orients the edges of the undirected skeleton graph by detection of v-structures and application of Meek's orientation rules. Outputs the CPDAG from skeleton.
 
-A usage example can be found in `benchmarks/benchmark_gpucsl.py`.
+Additional detail is found in the [API description](https://github.com/hpi-epic/gpucsl/blob/main/docs/Public-api.md).
+
+The following code snippet provides a small example for calling the `pc` function:
+```python
+import numpy as np
+from gpucsl.pc.pc import pc, DataDistribution
+from gpucsl.pc.helpers import correlation_matrix_of
+
+samples = np.random.rand(1000,10)
+max_level = 3
+alpha = 0.05
+correlation_matrix = correlation_matrix_of(samples)
```

Comment:
You are not forced to calculate the correlation matrix yourself . You can simply leave it and gpucsl calculates it for you. Or do you want to show something by probiding it?

Reply:
Good point, I removed it 👍